modin/experimental/core/storage_formats/hdk/query_compiler.py

```diff
@@ -557,18 +557,27 @@ def concat(self, axis, other, **kwargs):
 
     def drop(self, index=None, columns=None, errors: str = "raise"):
         if index is not None:
-            raise NotImplementedError("Row drop")
+            # Only column drop is supported by the HDK engine
+            raise NotImplementedError("Index drop")
```

Comment:
Don't you think that the previous version was more understandable? :)

Reply:
Removed.